runtime/moonbase/src/xcm_config.rs

```diff
@@ -272,7 +272,13 @@ impl xcm_executor::Config for XcmExecutorConfig {
 	// When we receive a non-reserve asset, we use AssetManager to fetch how many
 	// units per second we should charge
 	type Trader = (
-		UsingComponents<WeightToFee, SelfReserve, AccountId, Balances, DealWithFees<Runtime>>,
+		UsingComponents<
+			IdentityFee<Balance>,
+			SelfReserve,
+			AccountId,
+			Balances,
+			DealWithFees<Runtime>,
+		>,
```

Comment:
@girazoki this didn't cause any test failures but please review

Reply:
Oh, I was thinking this was test related, it needs to be the actual impl used by the runtime. I'll fix...

Reply:
Fixed in 482a9e4